Lombiq.HelpfulLibraries.LinqToDb/ITableAccessor.cs

```diff
@@ -23,6 +23,10 @@ ITable<T> GetTable<T>()
     /// Name of the YesSql collection that includes logically related objects. It is technically a prefix to the
     /// affected tables in the database.
     /// </param>
+    /// <remarks>
+    /// The reason for overloading the GetTable method instead of adding <paramref name="collectionName"/>
+    /// as an optional parameter is all the source code must be refactored where referenced.
```

Comment:
Please reference the error we've seen. I'm being particular about this because this is not trivial to readily see, as evidenced by the unexpected build errors.